src/main/java/org/sonarqube/gradle/SonarQubePlugin.java

```diff
@@ -112,7 +112,7 @@ private void configureTask(SonarQubeTask sonarQubeTask, Project project, Map<Str
       })
       .flatMap(List::stream)
       .collect(Collectors.toList());
-    sonarQubeTask.dependsOn(compileTasks);
+    sonarQubeTask.mustRunAfter(compileTasks);
```

Comment:
Even this change could deserve a separate ticket, since it may have impact on accuracy of Java analysis. Bytecode could be missing if people only run `gradle sonarqube`. Are you sure you want to do it?

Reply:
My bad, this is only for Android.